### ISSUE TITLE
Please explain ELI5 why "Block apps without internet" deserves a permanent notification warning

### ISSUE THREAD

**Issue by mailinglists35** [TrackerControl/tracker-control-android#436]
Isn't the whole point of VPN to prevent leaks going around the filtering? Why would I allow connections without vpn, if my goal is to block as much as possible all possible apps to all possible corners of the internet, and only allow legit traffic?

So if I'm thinking right, then how to configure the app to stop emiting this particular notification over and over again?

Thanks. and many thanks for developing and making this app free! I used to use noroot firewall but I got tired of whois-ing each request and deciding to allow or not.

**Comment by mailinglists35** [comment]
![Screenshot_20241123_004718_One UI Home](https://github.com/user-attachments/assets/207f9bac-6999-4fd9-b74c-47815c12777f)
yes I can dismiss it, but it will appear again whenever the app decides to.

**Comment by mailinglists35** [comment]
@kasnder ?!

**Comment by mailinglists35** [comment]
is this project abandoned? author alive?

**Comment by kasnder** [comment]
I don't think I can fix this issue, sorry!

**Comment by mailinglists35** [comment]
@kasnder please kindly carefully re-read my issue. I am asking for explanation, nothing more.
